<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MNK2190" failures="1" tests="5" name="TestNG test-3" time="0.054" errors="0" timestamp="2021-08-05T10:46:35 IST">
  <testcase classname="TestCase.Test_dependency" name="name1" time="0.003">
    <failure type="java.lang.AssertionError" message="name-1 fail">
      <![CDATA[java.lang.AssertionError: name-1 fail
at TestCase.Test_dependency.name1(Test_dependency.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- name1 -->
  <testcase classname="TestCase.Test_dependency" name="name3" time="0.012"/>
  <testcase classname="TestCase.Test_dependency" name="name4" time="0.003"/>
  <testcase classname="TestCase.Test_dependency" name="name5" time="0.004">
    <skipped/>
  </testcase> <!-- name5 -->
  <testcase classname="TestCase.Test_dependency" name="name2" time="0.0">
    <skipped/>
  </testcase> <!-- name2 -->
</testsuite> <!-- TestNG test-3 -->
